db/migrations/0003_alter_movie_actors_alter_movie_genres_and_more.py

```diff
@@ -0,0 +1,34 @@
+# Generated by Django 4.0.2 on 2024-02-06 19:26
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('db', '0002_cinemahall_movie_moviesession'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='movie',
+            name='actors',
+            field=models.ManyToManyField(related_name='movies', to='db.Actor'),
+        ),
+        migrations.AlterField(
+            model_name='movie',
+            name='genres',
+            field=models.ManyToManyField(related_name='movies', to='db.Genre'),
+        ),
+        migrations.AlterField(
+            model_name='moviesession',
+            name='cinema_hall',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='movie_sessions', to='db.cinemahall'),
+        ),
+        migrations.AlterField(
+            model_name='moviesession',
+            name='movie',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='movie_sessions', to='db.movie'),
+        ),
+    ]

```

Comment:
It is a good practice to include one migration per app per pull request in Django projects that represents all changes done to the models within this app. It allows us to prevent the `migrations` folder overload. Please, include one migration to your PR.